DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -1,4 +1,5 @@
-﻿@using Microsoft.Extensions.Configuration
+﻿@using DigitalLearningSolutions.Web.Helpers

```

Comment:
Just double checking whether these changes affect LearningPortal (e.g. /LearningPortal/Current) - the Learning Portal's navigation bar should not be getting all the new tabs (except for maybe My Account, which seems reasonable). Note that the LP user will never be signed out so including the sign in/registration buttons in the common layout won't be an issue!

Reply:
Ah, this is affecting the LP nav bar - I'll see if I can find a way around that.

Reply:
Whenever custom nav bar options are provided now, we won't show the Welcome, Find your centre, Pricing or Help links. Login and register/My account, switch application and Log out will still be shown depending on whether they're logged in.
Is that what you were looking for? The LearningPortal nav bar now looks like this:
![image](https://user-images.githubusercontent.com/30173746/114381641-3fe62300-9b83-11eb-956b-f2b7d01177ed.png)

Reply:
Yes, this is exactly what I was looking for (mainly just not affecting the LP functionality)! I think you're right in highlighting keeping the Help, because it was there before. 👍